crates/rrg/src/args.rs

```diff
@@ -75,9 +85,58 @@ fn parse_duration(value: &str) -> Result<Duration, String> {
     humantime::parse_duration(value).map_err(|error| error.to_string())
 }
 
+/// Decodes a Vector of hex digits to a Vector of byte values.
+fn hex_decode(hex: Vec<u8>) -> Result<Vec<u8>, HexDecodeError> {
```

Comment:
I understand that you named it after `hex::decode` but the convention for functions is that they usually start with a verb, so a more appropriate name would be `decode_hex` (and changing the function name should also affect name of the error struct).

Reply:
Done.